experimental/uefi/client/src/main.rs

```diff
@@ -50,10 +47,9 @@ async fn chat(client: &mut AttestationClient, message: String) -> anyhow::Result
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let cli = Args::parse();
 
-    let mut client =
-        AttestationClient::create(&cli.server, TEE_MEASUREMENT, Box::new(|_config| Ok(())))
```

Comment:
More placeholders that were not particularly meaningful, now this logic is centralized in the new trait (and probably we will have to add something to allow passing additional predicates at some point).